scripts/generate_openapi.py

```
@@ -0,0 +1,621 @@
+
+    evaluators = {}
+
+    # Find all evaluator definitions using regex
+    # Look for patterns like "evaluator/id": { ... }
+    evaluator_pattern = r'"([^"]+)":\s*{'
+
+    # Find all matches
+    matches = list(re.finditer(evaluator_pattern, content))
+    print(f"Found {len(matches)} evaluator matches")
+
+    for i, match in enumerate(matches):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)